### PR TITLE
doc: update README.md for proxy-mirror

### LINKED ISSUE #1847: doc: missing proxy-mirror plugin in README.md
we need to add the [doc](https://github.com/apache/incubator-apisix/blob/master/doc/plugins/proxy-mirror.md) of plugin to README.md.

### PR DESCRIPTION
**Pull request by scaat**
### What this PR does / why we need it:

Fix #1847

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
     - Hash-based Load Balancing: Load balance with consistent hashing sessions.
     - [Health Checks](doc/health-check.md): Enable health check on the upstream node, and will automatically filter unhealthy nodes during load balancing to ensure system stability.
     - Circuit-Breaker: Intelligent tracking of unhealthy upstream services.
-
+- [Proxy Mirror](doc/plugins/proxy-mirror.md): Provides the ability to mirror client requests.
```
Comment (moonming) [inline README.md:72]: need add sapces before `-`, keep line with above item.

File: README_CN.md [hunk 2]
```diff
@@ -68,7 +68,8 @@ A/B 测试、金丝雀发布(灰度发布)、蓝绿部署、限流限速、抵
     - 支持一致性 hash 的负载均衡：动态支持一致性 hash 的负载均衡。
     - [健康检查](doc/health-check.md)：启用上游节点的健康检查，将在负载均衡期间自动过滤不健康的节点，以确保系统稳定性。
     - 熔断器: 智能跟踪不健康上游服务。
-
+- [代理镜像](doc/zh-cn/plugins/proxy-mirror.md): 提供镜像客户端请求的能力。
```
Comment (moonming) [inline README_CN.md:71]: ditto

**Comment by membphis** [comment]
please take a look at the output of the CI:

```
.travis/openwhisk-utilities/scancode/scanCode.py --config .travis/ASF-Release.cfg ./
Scan detected 2 error(s) in 2 file(s):
Reading configuration file [.travis/ASF-Release.cfg]...
Scanning files starting at [./]...
Scan detected 2 error(s) in 2 file(s):
  [./README.md]:
      73: line has trailing whitespace.
  [./README_CN.md]:
      72: line has trailing whitespace.
```

**Comment by scaat** [comment]
files have been corrected.

**Pull request merged**